content/docs.md

```diff
@@ -149,7 +147,5 @@ Welcome to the developer documentation for OpenCensus. Here you can learn about
 [{{< sc_gloss1 >}}OPENCENSUS FOR GO gRPC DEVELOPERS{{< /sc_gloss1 >}}][2]
 
 
-[1]: /spanner
+[1]: https://medium.com/@orijtech/cloud-spanner-instrumented-by-opencensus-and-exported-to-stackdriver-6ed61ed6ab4e
```

Comment:
Should this be moved to blog.md instead?

Reply:
It is already in the blog. I think it is put here for visibility. There is a blurry distinction by this section and the blog. We will go through the entire website content soon to decide on the fate of such cases.

Reply:
SGTM